spec/moonshot/plugins/cleaner_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Moonshot::Plugins::Cleaner do
```

Comment:
This is in progress to write some proper tests for the class. Please ignore this for now.